packages/client/types/constEnums.ts

```diff
@@ -277,6 +282,14 @@ export const enum SubscriptionChannel {
   ORGANIZATION = 'organization'
 }
 
+export const enum SuggestedActionTypeEnum {
```

Comment:
`SuggestedActionTypeEnum` and `AuthIdentityTypeEnum` aren't being exported by Relay so I created them in the `constEnum.ts` file